scripts/python-maintainer/generate_python_package.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import pypi_cli
```

Comment:
pypi-cli hasn't been updated since 2015 (with the repo [archived by the owner](https://github.com/sloria/pypi-cli)). I would suggest using one of the libraries mentioned in the [pip user guide](https://pip.pypa.io/en/latest/user_guide/#using-pip-from-your-program) instead.

Reply:
I looked at distlib which looked most promising, however it seems to lack to offer the required information. As pypi_cli.py still works and the API seems stable since 2015, why not keep using it?

Reply:
It would help if you listed the information that you couldn't get with distlib.

Reply:
I did not try to hard (as the current version works just fine), but I did not figure out how to get the sha256sum instead of md5.

```
from distlib.locators import PyPIJSONLocator

locator = PyPIJSONLocator("https://pypi.python.org/pypi")

def get_package_info(name):
    info = {}
    package_info = locator.locate(name)
    info["version"] = package_info.metadata.version
    info["summary"] = package_info.metadata.summary
    info["digest"] = package_info.digests
    info["download_url"] = package_info.download_url

    print(info)

get_package_info("flask")
```

Reply:
You can get sha256 digests if you use `SimpleScrapingLocator("https://pypi.org/simple/")`. (It doesn't include summaries though.)

If/when https://bitbucket.org/pypa/distlib/pull-requests/42/check-for-a-digests-dictionary-before is merged, then `PyPIJSONLocator` should return sha256 digests as well.